src/main/kotlin/racingcar/domain/RacingCarGame.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain
+
+private const val MIN_RANDOM_VALUE = 0
+private const val MAX_RANDOM_VALUE = 9
+private const val THRESHOLD = 4
+
+class RacingCarGame(carCount: CarCount, roundCount: RoundCount) {
```

Comment:
변경되거나, 외부에서 사용되는 값으로 보이지는 않네요.

```suggestion
class RacingCarGame(private val carCount: CarCount, private val roundCount: RoundCount) {
```

Reply:
현재 `CarCount`와 `RoundCount` 는 초기화(init) 시 cars, rounds 프로퍼티를 생성하기 위해 사용되고 있으며, 
`RacingCarGame` 객체의 메서드에서는 사용하고 있지 않는데요. 

둘의 차이점으로는 `RacingCarGame` 객체의 메서드에서 사용 유무 인 것으로 알고 있어요.
혹시 제안주신 코드로 변경하면 얻는 장점이 있을까요? 🧐

Reply:
엇 요거는 제가 코드를 잘못 본 것 같습니다 ㅠㅠ

혼란스럽게 해드려서 죄송해요. 🙏